src/Compilers/Core/Portable/SourceGeneration/UserFunction.cs

```diff
@@ -52,9 +50,7 @@ internal static Action<TInput> WrapUserAction<TInput>(this Action<TInput> userAc
                 {
                     userAction(input);
                 }
-#pragma warning disable CS0618 // ReportIfNonFatalAndCatchUnlessCanceled is obsolete; tracked by https://github.com/dotnet/roslyn/issues/58375
-                catch (Exception e) when (FatalError.ReportIfNonFatalAndCatchUnlessCanceled(e))
-#pragma warning restore CS0618 // ReportIfNonFatalAndCatchUnlessCanceled is obsolete
+                catch (Exception e) when (ExceptionUtilities.IsNotCancelled(e))
```

Comment:
maybe WrapUserAction should take a CT?

Reply:
These changes are no longer part of this PR.